src/components/footer/Footer.js

```diff
@@ -5,20 +5,22 @@ import Button from 'components/button/Button';
 
 // @TODO: Bring it back when ready
 // import DonateButton from './DonateButton';
-import styles from './footer.module.scss';
 
-function Footer({ onClickLegal, t }) {
+function Footer({ onClickLegal }) {
   return (
-    <footer className={styles.footerMain}>
-      <div className={styles.footerInner}>
-        <div className={styles.column}>{/* <DonateButton /> */}</div>
+    <footer className={'w-full border-t border-secondary-lightest'}>
+      <div
+        className={
+          'flex flex-col md:flex-row justify-center items-center sm:justify-end flex-wrap pt-4 pb-4 pr-8 pl-8 mr-16 ml-16 '
```

Comment:
Would be nice to write this in more than 1 line so we don't have more than 80 chars per line